hydroflow_plus_test/src/cluster/paxos.rs

```diff
@@ -765,11 +783,11 @@ fn p_p2b<'a, P: PaxosPayload>(
         } else {
             None
         }))
-        .fold_keyed(
+        .fold_keyed_commutative(
             q!(|| (0, None)),
             q!(|accum, value| {
                 accum.0 += 1;
-                accum.1 = Some(value);
+                accum.1 = Some(value); // TODO(shadaj): this is not commutative?
```

Comment:
Addressing the TODO: It's fine that it's not commutative because Paxos invariants say that any p2b where max ballot = the proposer's current ballot must have the same value